updater/internal/rollback/rollback.go

```diff
@@ -140,7 +153,7 @@ func backupFiles(logger *zap.Logger, installDir, outputPath string) error {
 		}
 
 		// Fail if copying the input file to the output file would fail
-		if err := file.CopyFile(logger.Named("copy-file"), inPath, outPath, false); err != nil {
+		if err := file.CopyFile(logger.Named("copy-file"), inPath, outPath, false, false); err != nil {
 			return fmt.Errorf("failed to copy file: %w", err)
 		}
```

Comment:
I know this doesn't matter (see above comment about `useInFilePermBackup` = true and `overwrite` = false), but I'd expect `useInFilePermBackup` to be set to true here when backing up a file.